.github/workflows/dotnet.yml

```diff
@@ -38,10 +38,17 @@ jobs:
       with:
         distribution: temurin
         java-version: 11
-    - name: Build
-      run: dotnet build --no-restore
-    - name: Test
+    - name: Build .NET Framework Project
+      run: dotnet build dotnet-framework/lowkey-vault-example-dotnet.sln --no-restore
```

Comment:
Hi @irby ,
thank you for spending time on this project and opening this PR! It was much needed that someone more familiar with .Net taken a look and give feedback! 

Should we do the same rename/move in Line 32 as well?
```  - run: nuget restore dotnet-framework/lowkey-vault-example-dotnet.sln```

Also, If I get it right, we have two modules after this PR is merged, but we are running tests on one of them as it is now. Should there be an additional line for dotnet-core after line 53?
```dotnet test dotnet-core/lowkey-vault-example-dotnet-core.sln --no-build --verbosity normal```

What do you think?

Reply:
Whoops, you're correct. I'll add that change in soon.

Reply:
Thank you!